cmd/system-probe/config/config.go

```diff
@@ -182,42 +165,36 @@ func load(configPath string) (*Config, error) {
 		cfg.Set(key(spNS, "max_conns_per_message"), c.MaxConnsPerMessage)
 	}
 
-	// this check must come first so we can accurately tell if system_probe was explicitly enabled
+	// this check must come first, so we can accurately tell if system_probe was explicitly enabled
 	npmEnabled := cfg.GetBool("network_config.enabled")
 	usmEnabled := cfg.GetBool("service_monitoring_config.enabled")
 
 	if npmEnabled {
-		log.Info("network_config.enabled detected: enabling system-probe with network module running.")
 		c.EnabledModules[NetworkTracerModule] = struct{}{}
 	} else if c.Enabled && !cfg.IsSet("network_config.enabled") && !usmEnabled {
 		// This case exists to preserve backwards compatibility. If system_probe_config.enabled is explicitly set to true, and there is no network_config block,
 		// enable the connections/network check.
-		log.Info("network_config not found, but system-probe was enabled, enabling network module by default")
+		log.Info("`system_probe_config.enabled` is deprecated, enable NPM with `network_config.enabled` instead")
 		c.EnabledModules[NetworkTracerModule] = struct{}{}
 		// ensure others can key off of this single config value for NPM status
 		cfg.Set("network_config.enabled", true)
 		npmEnabled = true
 	}
 
 	if !npmEnabled && usmEnabled {
-		log.Info("service_monitoring.enabled detected: enabling system-probe with network module running.")
```

Comment:
perhaps you may want to keep this in one release?

Reply:
These logs got moved to system-probe at runtime, so they don't get logged for other processes that consume the system-probe config.